src/main/kotlin/no/nav/syfo/varsel/arbeidstaker/ArbeidstakerVarselApi.kt

```diff
@@ -31,12 +31,10 @@ fun Route.registerArbeidstakerVarselApi(
                 val requestPersonIdent = call.personIdent()
                     ?: throw IllegalArgumentException("No PersonIdent found in token")
 
-                val dialogmoteDTOList = dialogmoteService.getDialogmoteList(
-                    personIdentNumber = requestPersonIdent
-                ).map { dialogmote ->
-                    dialogmote.toDialogmoteDTO()
-                }
-                call.respond(dialogmoteDTOList)
+                val arbeidstakerVarselDTOList = dialogmoteService.getDialogmoteList(
+                    personIdentNumber = requestPersonIdent,
+                ).toArbeidstakerVarselDTOList()
```

Comment:
Burde vi ha hentet varslene bare fra det *aktive* møtet?  (dvs i stedet for å hente fra alle møtene)

Reply:
Godt spørsmål!
Jeg tok ikke stilling til noen form for filtrering her nå. Tenkte litt på det, men vet ikke helt hva eSYFO ønsker å vise, mtp aktiv, historikk e.l.